blogposts/2022/release-3.36.md

```diff
@@ -0,0 +1,60 @@
+    url: https://github.com/sourcegraph/sourcegraph/issues/17879
+    category: Batch Changes
+  - description: 'Notebooks are now fully persistable and can be shared via a unique URL. We support two visibility modes: private (only the creator can view the notebook) and public (everyone can view the notebook). This is a `Beta` feature and may change in following releases.'
+    url: https://github.com/sourcegraph/sourcegraph/issues/27384
+    category: Search
+  - description: 'Search results across repositories are now ordered by repository rank to make results more relevant. By default the rank is the number of stars a repository has. An administrator can inflate the rank of a repository via `experimentalFeatures.ranking.repoScores`. If you notice increased latency in results, you can disable this feature by setting `experimentalFeatures.ranking.maxReorderQueueSize` to 0.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/29856
+    category: Search
+  - description: 'Symbolic links are now searchable to make files easier to find. Previously it was possible to navigate to symbolic links in the repository tree view, however the symbolic links were ignored during searches.'
+    url: https://github.com/sourcegraph/zoekt/pull/237
+    category: Search
+  - description: "Sourcegraph's API (streaming search, GraphQL, etc.) may now be used from any domain when using an access token for authentication, or with no authentication in the case of Sourcegraph.com."
+    url: https://github.com/sourcegraph/sourcegraph/pull/28775
+    category: API
+  - description: 'The endpoint `/search/stream` will be retired in favor of `/.api/search/stream`. This requires no action unless you have developed custom code against `/search/stream`. We will support both endpoints for a short period of time before removing `/search/stream`. Please refer to the documentation for more information.'
+    url: https://docs.sourcegraph.com/api/stream_api
+    category: API
+---
+
+Sourcegraph 3.36 is now available! Here are some highlights from this release:
+
+## Push new branches from Batch Changes to forks
+
+Before this release, creating a batch change required you (or your service-account) to have write-access to every target repository. This was incompatible with having tight repository access control, and was impractical at a large scale; you want all users to be able to create batch changes on all repositories, but also not grant them overly broad write permissions.
+
+That's why we are introducing [batch changes on forks](https://docs.sourcegraph.com/admin/config/batch_changes), which makes it possible for branches created by Batch Changes to be pushed to a fork of the repository rather than the repository itself. This solves those access control challenges and lets you safely enable Batch Changes for everyone. In Sourcegraph 3.36, this is an instance setting that can be turned either on or off for all users, and is off by default.
```

Comment:
The specific setting to configure this appears in the changelog, so I think a deep-link is good here. 

Also, re: capitalization, yes — I believe caps here makes it more consistent since in this sentence we are introducing this as a product, not talking about the "batch changes" that are part of that product

```suggestion
That's why we are introducing [Batch Changes on forks](https://docs.sourcegraph.com/admin/config/batch_changes#forks), which makes it possible for branches created by Batch Changes to be pushed to a fork of the repository rather than the repository itself. This solves those access control challenges and lets you safely enable Batch Changes for everyone. In Sourcegraph 3.36, this is an instance setting that can be turned either on or off for all users, and is off by default.
```